Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -520,8 +529,6 @@ - (void)update
     [self refreshTitle];
     [self refreshSearchBar];
 
-    BOOL isTrashOnScreen = self.isDeletedFilterActive;
```

Comment:
This was a warning in the project that I probably caused recently, but I was tired of it staring at me.  So tossing that in here to shut it up.